integration_test/third_party_apps_data/applications/couchbase/debian_ubuntu/install

```diff
@@ -13,8 +13,11 @@ sudo dpkg -i ./couchbase-release-1.0-amd64.deb
 sudo apt-get update
 sudo apt-get install -y couchbase-server-community=7.1.0-2556-1
 
+# Wait for the app to boot
+sleep 60
+
 /opt/couchbase/bin/couchbase-cli cluster-init -c 127.0.0.1 --cluster-username admin --cluster-password password \
   --cluster-name otelc --cluster-ramsize 256 --cluster-index-ramsize 256 --services data,index,query,fts \
   --index-storage-setting default
 
-/opt/couchbase/bin/couchbase-cli bucket-create -c 127.0.0.1 --username otelu --password otelpassword --bucket-type couchbase --bucket-ramsize 256 --bucket otelb
\ No newline at end of file
+/opt/couchbase/bin/couchbase-cli bucket-create -c 127.0.0.1 --username admin --password password --bucket-type couchbase --bucket-ramsize 128 --bucket otelb
```

Comment:
Not an expert with Couchbase, but in my local experiment, I got permission denied before changing the username and password to match the cluster-username and cluster-password above.

Reply:
Also hit a `bucket ramsize too big` error. Reducing it to 128 worked.